index.html

```diff
@@ -314,38 +314,24 @@ <h2>The <dfn>requestIdleCallback()</dfn> method</h2>
         <a>idle callback identifier</a>.</li>
         <li>Push <var>callback</var> to the end of <var>window</var>'s <a>list
         of idle request callbacks</a>, associated with <var>handle</var>.</li>
-        <li>Return <var>handle</var> and then continue running this algorithm
-        asynchronously.
-          <p class="note">The following steps run in parallel and queue a timer
-          similar to <code>setTimeout()</code> if the optional <code>timeout</code> property is
-          provided. From here, the idle and timeout callbacks are raced and
-          cancel each other—e.g. if the idle callback is scheduled first, then
-          it cancels the timeout callback, and vice versa.</p>
-        </li>
         <li>If the <code><dfn data-dfn-for="IdleRequestOptions">timeout</dfn>
         </code> property is present in <var>options</var> and has a positive
-        value:
+        value, then <a>run steps after a timeout</a>, given <var>window</var>,
+        "<code>requestIdleCallback-timeout</code>", <var>timeout</var>, and the
+        following step:
           <ol>
-            <li>Wait for <var>timeout</var> milliseconds.</li>
-            <li>Wait until all invocations of this algorithm, whose
-            <var>timeout</var> added to their posted time occurred before this
-            one's, have completed.</li>
-            <li>Optionally, wait a further user-agent defined length of time.
```

Comment:
I don't think we should drop this UA-defined flexibility

Reply:
"Run steps after a timeout" still gives you that flexibility.

Reply:
OK, that's fair. I guess it would also enable UAs applying different padding strategies for rIC vs. non-rIC timeouts